src/common/utils/log/logger.go

```diff
@@ -66,11 +76,20 @@ func New(out io.Writer, fmtter Formatter, lvl Level, options ...interface{}) *Lo
 			depth = d
 		}
 	}
+
+	mu, ok := writerMutexMap[out]
```

Comment:
What is this used for?

Reply:
This is used to lock the output writer, when multi logger use the same output writer.

En, this is may not needed for the is a root logger in the system (the default logger), and other loggers are clone from this. I will remove the `writerMutexMap`.